data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/MarbleZone.meta

```diff
@@ -1,3 +1,4 @@
 Sonic the Hedgehog - Marble Zone
 0x03
-bgm
\ No newline at end of file
+bgm
+DodongosCavern,FireTemple,Dungeon,AncientDungeon
```

Comment:
The location line should have at least one group from each level of specificity (exact, high, mid and low).